homeassistant/components/verisure/coordinator.py

```diff
@@ -43,8 +46,11 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
     async def async_login(self) -> bool:
         """Login to Verisure."""
         try:
-            await self.hass.async_add_executor_job(self.verisure.login)
-        except VerisureError as ex:
+            await self.hass.async_add_executor_job(self.verisure.login_cookie)
+        except VerisureLoginError as ex:
+            LOGGER.error("Could not log in to verisure, %s", ex)
```

Comment:
I don't think we need to log an error here if we raise an exception. Same goes for all the cases here.